plotsearch/models/plot_search.py

```diff
@@ -574,3 +583,4 @@ class Meta:
 auditlog.register(PlotSearch)
 auditlog.register(InformationCheck)
 auditlog.register(RelatedPlotApplication)
+auditlog.register(AreaSearch)
```

Comment:
This and the change above should not be in this pull request, so I will remove it